docs/run_test/platform.rst

```diff
@@ -9,7 +9,69 @@ Run tests on different platforms
 Run on Azure
 ------------
 
-Follow the same procedure in :doc:`Getting started with Azure <quick_run>`.
+VM's can be deployed on Azure using images from vhd, shared image
+gallery or marketplace. If multiple types are specified, the first
+non-empty type is picked in the following order :
+vhd, shared image gallery and marketplace.
+
+Running using vhd
+^^^^^^^^^^^^^^^^^
+To run using vhd, add the following to runbook :
+.. code:: yaml
+   platform:
+   - type: azure
+      ...
+      requirement:
+         ...
+         azure:
+            ...
+            vhd: "<VHD URL>"
+
+The `<VHD URL>` can be a `SAS URL`.
```

Comment:
Add explanation that, the sas url cannot be deployed directly. It will be copied before deploying.

Reply:
I saw the explanation in comments in platform.py but decided to skip it as it is was an implementation detail. How will it be useful for the user?

Reply:
The copied vhd needs to be deleted by user for now. LISA has no utility to help clean up it. Another thing is it may take longer time to copy the vhd. The test run will be slower.

Reply:
Got it! Thanks for the explanation. I will mention these implications in the doc.